### ISSUE TITLE
macOS user agents incorrectly contain '; U'

### ISSUE THREAD

**Issue by feythr** [jake-cryptic/AbsoluteDoubleTrace#3]
The macOS user agents shouldn't have `; U` in the middle of them:

https://github.com/jake-cryptic/AbsoluteDoubleTrace/blob/897c75984c3f750e51f1c4feaf31a1de584d3fb3/MyTrace/js/script.background.js#L187-L192

For example, my unaltered user agent is:
```
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36
```

In addition, it would be great if we could expand the list of macOS versions (at the very least to contain the latest `10_14_1`). FWIW, I find https://techblog.willshouse.com/2012/01/03/most-common-user-agents/ to be a great resource for common user agents.

From that list, common versions to add would be:
- `10_13_6`
- `10_14`
- `10_14_0`
- `10_12_6`
- `10_14_1`

**Comment by jake-cryptic** [comment]
Thanks for bringing this to my attention! I shall fix it in the next few hours :) and also I will take a look at the resource too.

I'm planning on adding the ability to have a custom list of user agents in a future version, so stay tuned for that.

**Comment by jake-cryptic** [comment]
This should all be fixed in 2.0.5 now which I just pushed out to Chrome and Firefox.
I also added in a special case for Firefox on Mac as per your suggestion.

Thanks for reporting all this :)
